### ISSUE TITLE
where should `db_admin_password` be set?

### ISSUE THREAD

**Issue by squarepegsys** [sovereign/sovereign#703]
I get this error:

``fatal: [sovereign.hostetlerhome.com]: FAILED! => {"failed": true, "msg": "the field 'args' has an invalid value, which appears to include a variable that is undefined. The error was: 'db_admin_password' is undefined\n\nThe error appears to have been in '/Users/mikeh/Projects/sovereign/roles/common/tasks/postgres.yml': line 12, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Set password for PostgreSQL admin user\n  ^ here\n"}```

I see in `roles/mailserver/defaults/main.yml` it's set like this:

```db_admin_password: "{{ lookup('password', secret + '/' + 'db_admin_password', length=32) }}"```

But I didn't have a `secret/db_admin_password` file generated so I created my own but I get the same error.

Any hints on what do to?

**Comment by neuhaus** [comment]
I've seen that issue also. Need to investigate.

**Comment by neuhaus** [comment]
Does the problem go away if you run sovereign multiple times?

**Comment by squarepegsys** [comment]
Yeah I've ran it over and over again on the same server. I'm willing to try it again, on a vanilla server, again.

**Comment by grahamas** [comment]
The intro still lacks a clear explanation of the format expected for secrets
